public/md/index.md

```diff
@@ -15,3 +15,26 @@ Here are a few reasons why taking part in algorithmics contests is a good idea:
 - You (sometimes) receive **balloons** when you successfully complete a problem.
 - Karaoke. Need I say more?
 - It's a perfect addition to your **Tinder** profile!
+
+## What is ICPC
+ICPC stands for International Collegiate Programming Contest. It is an algorithmic programming contest for college students. Teams of three, representing their university, work to solve the most real-world problems, fostering collaboration, creativity, innovation, and the ability to perform under pressure.
+
+## How it works ? 
+There is multiple levels of competition. Here in Belgium there is 3 levels:
+- BAPC
+- NWERC
+- Worlds Finals
+
+### BAPC
+BAPC stands for Benelux Algorithm Programming Contest
```

Comment:
```suggestion
### preBAPC & BAPC

BAPC (Benelux Algorithm Programming Contest).
```

Ce serait cool d'avoir un peu plus à dire là dessus mais j'imagine que you're on it :)